cmd2/cmd2.py

```diff
@@ -3712,29 +3743,23 @@ def do_set(self, args: argparse.Namespace) -> None:
         if args.param:
             try:
                 settable = self.settables[args.param]
+                if settable.settable_obj is None:
```

Comment:
Why would settable_obj be None?

Reply:
The Settable class doesn't require the settable_obj to be set. For legacy purposes, anything without a defined object is assumed to be the the thing the settable is added to.

Reply:
Thinking about this more, this is a lot less likely since I 'broke' the existing API. However I think this will still be necessary in case people don't use the `add_settable` function.